newrelic/hooks/framework_django.py

```diff
@@ -420,6 +416,36 @@ def _nr_wrapper_BaseHandler_get_response_(wrapped, instance, args, kwargs):
     return response
 
 
+async def _nr_wrapper_BaseHandler_get_response_async_(wrapped, instance, args, kwargs):
+    response = await wrapped(*args, **kwargs)
+
+    if current_transaction() is None:
+        return response
+
+    request = _bind_get_response(*args, **kwargs)
```

Comment:
Should we swap this to a `bind_args` call and drop the `_bind_get_response` function definition in this file?